FusionIIIT/applications/health_center/models.py

```diff
@@ -27,6 +28,7 @@ class Doctor(models.Model):
     doctor_phone = models.CharField(max_length=10)
     specialization = models.CharField(max_length=100)
     active = models.BooleanField(default=True)
+    contact_no = models.CharField(max_length=10, validators=[MinLengthValidator(10)], default=0000000000)
```

Comment:
The default must be a string for a CharField. Use min_length=10 instead of the validator.

Reply:
That is used for forms.CharField() and not for models.CharField(). That is why I used a validator.

Reply:
you need to change the default , it must be string

Reply:
@samay-rgb PTAL.

Reply:
Migration file should be updated too.